Xamarin.PropertyEditing/Properties/Resources.resx

```diff
@@ -577,4 +578,172 @@
     <value>Select {0} type</value>
     <comment>Select {property} type</comment>
   </data>
+  <data name="PropertyFilterLabel" xml:space="preserve">
+    <value>Property Filter</value>
+    <comment>Property Filter placeholder text</comment>
+  </data>
+  <data name="PropertyColumnTitle" xml:space="preserve">
+    <value>Property</value>
+    <comment>Title text for property column</comment>
+  </data>
+  <data name="ValueColumnTitle" xml:space="preserve">
+    <value>Value</value>
+    <comment>Title text for value column</comment>
+  </data>
+  <data name="AccessibilityNumeric" xml:space="preserve">
+    <value>{0} Numeric Editor</value>
+    <comment>Editor for Numeric Value</comment>
+  </data>
+  <data name="AccessibilityBoolean" xml:space="preserve">
+    <value>{0} Boolean Editor</value>
+    <comment>Editor for Boolean Value</comment>
+  </data>
+  <data name="AccessibilityCombobox" xml:space="preserve">
+    <value>{0} Combobox Editor</value>
+    <comment>Editor for Combobox Value</comment>
+  </data>
+  <data name="AccessibilityCollection" xml:space="preserve">
+    <value>{0} property, edit collection</value>
+    <comment></comment>
+  </data>
+  <data name="AccessibilityCollectionHelp" xml:space="preserve">
+    <value>Opens a window to edit the collection</value>
+  </data>
+  <data name="AccessibilityCollectionTypeSelector" xml:space="preserve">
+    <value>Type list</value>
+  </data>
+  <data name="AccessibilityCollectionAddButton" xml:space="preserve">
+    <value>Add instance of type</value>
+  </data>
+  <data name="AccessibilityCollectionRemoveButton" xml:space="preserve">
+    <value>Remove instance</value>
+  </data>
+  <data name="AccessibilityString" xml:space="preserve">
+    <value>{0} String Editor</value>
+    <comment>Editor for String Value</comment>
+  </data>
+  <data name="AccessibilityChar" xml:space="preserve">
+    <value>{0} Char Editor</value>
+    <comment>Editor for Char Value</comment>
+  </data>
+  <data name="AccessibilityTimeSpan" xml:space="preserve">
+    <value>{0} TimeSpan Editor</value>
+    <comment>Editor for TimeSpan Value</comment>
+  </data>
+  <data name="AccessibilityDateTime" xml:space="preserve">
+    <value>{0} DateTime Editor</value>
+    <comment>Editor for DateTime Value</comment>
+  </data>  
+  <data name="AccessibilityXEditor" xml:space="preserve">
+    <value>{0} X Editor</value>
+    <comment>Editor for X Value</comment>
+  </data>
+  <data name="AccessibilityYEditor" xml:space="preserve">
+    <value>{0} Y Editor</value>
+    <comment>Editor for Y Value</comment>
+  </data>
+  <data name="AccessibilityWidthEditor" xml:space="preserve">
+    <value>{0} Width Editor</value>
+    <comment>Editor for Width Value</comment>
+  </data>
+  <data name="AccessibilityHeightEditor" xml:space="preserve">
+    <value>{0} Height Editor</value>
+    <comment>Editor for Height Value</comment>
+  </data> 
```

Comment:
The comment's here should reflect how the string in the `value` element should be populated. We can give a few examples if needed.

In this case does it populate as `Margin Height Editor`? `Editor for Height Value` is a bit weird as a comment as it does not relate to the real string.